src/components/forms/TextField.js

```diff
@@ -60,11 +55,19 @@ export default {
       const inputLength = (this.inputValue && this.inputValue.toString() || '').length
       let min = inputLength
 
-      if (this.min !== 0 && inputLength < this.min) {
-        min = this.min
+      if (this.counterMin !== 0 && inputLength < this.counterMin) {
+        min = this.counterMin
       }
 
-      return `${min} / ${this.max}`
+      return `${min} / ${this.counterMax}`
+    },
+    counterMin () {
+      const parsedMin = Number.parseInt(this.min, 10)
+      return Number.isNaN(parsedMin) ? 0 : parsedMin
+    },
+    counterMax () {
+      const parsedMax = Number.parseInt(this.max, 10)
+      return Number.isNaN(parsedMax) ? 25 : parsedMax
```

Comment:
@johnleider See above. `counterMin` and `counterMax` will always return integers.